WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -179,7 +179,8 @@ - (void)viewWillDisappear:(BOOL)animated
     [super viewWillDisappear:animated];
 
     if (self.isStandalone) {
-        if (!self.isStandaloneEditorDismissingAfterSave) {
+        if ((self.isBeingDismissed || self.parentViewController.isBeingDismissed) && !self.isStandaloneEditorDismissingAfterSave) {
```

Comment:
This is not great, but, it's an improvement over the production version. I added a stretch task to replace this with either ViewModel or [child contexts](https://www.kodeco.com/7586-multiple-managed-object-contexts-with-core-data-tutorial/page/3).

Reply:
I'm going to give the child contexts idea another try, but this should work for now.